OpenOversight/app/main/views.py

```diff
@@ -730,6 +732,9 @@ def add_department():
                         )
                         order += 1
                 db.session.commit()
+                Department(id=department.id).remove_database_cache_entries(
+                    [KEY_DEPTS_BY_STATE],
+                )
```

Comment:
Remove cache entry for departments by state when adding a department.

Reply:
There's an argument for only clearing it when you modify the state, but I don't think it's worth the extra control statements.